src/test/regress/sql/functional_deps.sql

```diff
@@ -1,5 +1,8 @@
 -- from http://www.depesz.com/index.php/2010/04/19/getting-unique-elements/
 
+-- Test functional dependencies with standard planner
+SET optimizer = off;
```

Comment:
Why not just add a file called `functional_deps_optimizer.out` with the output for GPORCA, and leave the output for the standard planner in that file?

Reply:
For 2 reasons:
1) I wasn't aware about this feature (to have a separate output file when GPORCA is used) 😄 
2) If we add a separate out file for GPORCA, that means that all newly added queries will also be tested with postgres planner when called with ```optimizer=off```.
Right now I do not like this idea, because of https://tracker.yandex.ru/ADBDEV-4888
Postgres optimizer might output column values in grouped rows, that actually cannot be uniquelly identified.
For ex., from this ticket description:
```
SELECT count(*), a, b FROM t GROUP BY GROUPING SETS ((a), ()) ORDER BY a;
 count | a |   b   
-------+---+-------
     1 | 1 | text1
     1 | 2 | text2
     1 | 3 | text3
     1 | 4 | text4
     1 | 5 | text5
     5 |   | text5
(6 rows)
```
the last row is the grouping by all rows. The 'b' should be empty. But with postgres planner the output is filled with the value from one of the rows.
It is a bug. And I can't predict if this value will be constant from launch to launch (simply because I do not know the root cause). This test might become flaky if we will be testing all newly added queries with postgres planner until ADBDEV-4888 is resolved. Also, it means that  for postgres optimizer case we will need to add expected output with known errors.

Reply:
> 2\. If we add a separate out file for GPORCA, that means that all newly added queries will also be tested with postgres planner when called with `optimizer=off`.

You can use the explicit optimizer only for new tests by adding TODO to disable it.

Reply:
ok